manager-bundle/tests/Command/MaintenanceModeCommandTest.php

```diff
@@ -37,6 +38,9 @@ protected function tearDown(): void
     public function testEnable(string $expectedTemplateName, array $expectedTemplateVars, string|null $customTemplateName = null, string|null $customTemplateVars = null): void
     {
         $twig = $this->mockEnvironment();
+
+        $this->mockTwigLoader($twig, $expectedTemplateName);
```

Comment:
Why does this have to be a separate method? It is not used anywhere else.

Reply:
I originally tested it differently, thus moved it away.

fixed in 82709a53162bee39efef0c6708c42867e632db24